### ISSUE TITLE
Scoresheet tab order breaks with panels of seven

### ISSUE THREAD

**Issue by czlee** [czlee/tabbycat#127]
The tab order for the "save" box is set to 100, on the assumption that the adjudicator score fields will never go that high. This breaks for the sixth adjudicator. Ideally, the tab order should be dynamically set according to the number of adjudicators on the panel. If this is too hard, it should be set to 1000 or something like that.
